test/NuGet.CommandLine.Test/RestoreProjectJsonTest.cs

```diff
@@ -9,6 +10,418 @@ namespace NuGet.CommandLine.Test
     public class RestoreProjectJsonTest : IDisposable
     {
         [Fact]
+        public void RestoreProjectJson_FloatReleaseLabelHighestPrelease()
+        {
+            // Arrange
+            var tempPath = Path.GetTempPath();
+            var workingPath = Path.Combine(tempPath, Guid.NewGuid().ToString());
+            var repositoryPath = Path.Combine(workingPath, Guid.NewGuid().ToString());
+            var currentDirectory = Directory.GetCurrentDirectory();
```

Comment:
I think it's easier to debug when they are in the same working folder, but either would work.